src/bridge/clipboard.rs

```diff
@@ -0,0 +1,44 @@
+use std::error::Error;
+
+use rmpv::Value;
+
+use clipboard::ClipboardContext;
+use clipboard::ClipboardProvider;
+
+pub fn get_remote_clipboard() -> Result<Value, Box<dyn Error>> {
+    let mut ctx: ClipboardContext = ClipboardProvider::new()?;
+    let lines = ctx
+        .get_contents()?
+        .replace("\r", "")
+        .split("\n")
```

Comment:
I think this line ending conversion should be configurable, since it depends on both the source and host OS, but there could be some autodetected defaults. 

Also currently it looks like it converts from Windows endings to Linux endings, when pasting pasting into Neovim, but when copying from Neovim, it does not not do the opposite. That is different from how the default `win32yank.exe` implementation works on WSL, which converts both ways https://github.com/neovim/neovim/blob/22b538139606321bb1083665c510e81adb3d6670/runtime/autoload/provider/clipboard.vim#L130

Reply:
Fair points. Should we look for current file endline encoding and make it compatible with that?

get clipboard: check current endline encoding, convert?
set clipboard: no furthur changes, direct code copy

Reply:
I'm not sure to be honest. 

I'm note sure, but I think nvim always internally uses `LF`, so stripping the `CR` like you do here should be fine for pasting, otherwise I guess the current file encoding is a better option. 

For copying you probably want to use the convention of the operating system in which Neovide is running, so `CR LF` on Windows.

But ideally it should be configurable, and even configurable at runtime. There has been a few instances where I have wished that I could copy in the Linux format. So for people that needs that in a common scenario could even make a special keybinding. But I think it's fine to just use a config variable for this, since you can map a custom function that sets the variable, copies  and then resets it.

Reply:
I think this is the way to go for now. If in the future we find a way to make it work automatically, no reason we can't fix it later. One bit though: the default for the setting should be set by platform